src/main/java/com/redhat/cloud/notifications/models/BehaviorGroupAction.java

```diff
@@ -25,6 +25,7 @@
     @ManyToOne
     @MapsId("behaviorGroupId")
     @JoinColumn(name = "behavior_group_id")
+    @JsonIgnore
```

Comment:
This was causing a stack overflow when trying to serialize the `BehaviorGroup` ( BehaviorGroup -> BehaviorGroupAction -> BehaviorGroup -> ...)